api/datadogV1/model_add_signal_to_incident_request.go

```diff
@@ -166,8 +166,20 @@ func (o *AddSignalToIncidentRequest) UnmarshalJSON(bytes []byte) (err error) {
 		o.UnparsedObject = raw
 		return nil
 	}
+	additionalProperties := make(map[string]interface{})
+	if err = json.Unmarshal(bytes, &additionalProperties); err == nil {
+		delete(additionalProperties, "add_to_signal_timeline")
+		delete(additionalProperties, "incident_id")
+		delete(additionalProperties, "version")
```

Comment:
Can you write a helper so that we can do something like `DeleteKeys(additionalProperties, []string{"add_to_signal_timeline", "incident_id", "version"})`?